packages/driver/cypress/e2e/commands/actions/trigger.cy.ts

```diff
@@ -1280,7 +1282,7 @@ describe('src/cy/commands/actions/trigger', () => {
           const { lastLog } = this
           const { fromElWindow } = Cypress.dom.getElementCoordinatesByPosition($btn)
 
-          expect(lastLog.get('coords')).to.deep.eq(fromElWindow, 'x', 'y')
+          expect(lastLog.get('coords')).to.deep.eq(fromElWindow)
```

Comment:
This doesn't accept more than 1 argument and what even were these extra arguments doing - nothing...